site/en/r2/tutorials/distribute/multi_worker.ipynb

```diff
@@ -268,69 +268,74 @@
       "outputs": []
     },
     {
+      "cell_type": "markdown",
       "metadata": {
         "id": "P94PrIW_kSCE",
         "colab_type": "text"
       },
-      "cell_type": "markdown",
       "source": [
-        "Note that while the learning rate is fixed in this example, in general it may be necessary to adjust the learning rate based on the global batch size."
+        "Note: Although the learning rate is fixed in this example, in general it may be necessary to adjust the learning rate based on the global batch size."
       ]
     },
     {
+      "cell_type": "markdown",
       "metadata": {
         "colab_type": "text",
         "id": "UhNtHfuxCGVy"
       },
-      "cell_type": "markdown",
       "source": [
-        "## MultiWorkerMirroredStrategy\n",
+        "## `MultiWorkerMirroredStrategy`\n",
```

Comment:
No code in headers. There are formatting issues across github/colab/site

Reply:
Ah, good to know.